client/components/data/query-user-devices/index.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { PropTypes, Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestUserDevices } from 'state/user-devices/actions';
+import { getUserDevices } from 'state/selectors';
+
+class QueryUserDevices extends Component {
+	static propTypes = {
+		requestUserDevices: PropTypes.func
+	}
+
+	componentDidMount() {
+		if ( this.props.devices === null ) {
```

Comment:
What happens if we want fresh data after the first load?

I think we should be able to call `this.props.requestUserDevices();` without this check.